flask_security/forms.py

```diff
@@ -203,8 +204,9 @@ def validate(self):
 class LoginForm(Form, NextFormMixin):
     """The default login form"""
 
-    email = StringField(get_form_field_label('email'))
```

Comment:
I don't thinks we need it. But it looks like deleting it will break some tests since they will test the responses are matched to these error strings when invalid email or password form data are posted.